clientv3/clientv3util/example_key_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2017 The etcd Authors
```

Comment:
CI failed due to compilation issue.
```
gopath/src/github.com/coreos/etcd/clientv3/clientv3util/example_key_test.go:53: undefined: endpoints
gopath/src/github.com/coreos/etcd/clientv3/clientv3util/example_key_test.go:54: undefined: dialTimeout
```

you still need to move this file to clientv3 dir. so `endpoints` and `dialTimeout` will be defined.